frontend/src/App.js

```diff
@@ -3,11 +3,21 @@ import { Fragment } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import "semantic-ui-css/semantic.min.css";
 import './App.css';
+import React, { PureComponent } from 'react';
+import { PieChart, Pie, Sector, Cell, ResponsiveContainer } from 'recharts';
 // Pages Import
-
 import HomePage from "./Pages/Homepage"
 import Footer from "./Components/Footer/footer"
 const App = () => {
+ 
+  const data01 = [
```

Comment:
For fake/Test_data , You can create new file on `frontend/src/Test_data/` and fetch from there!